apps/hash-frontend/src/components/remote-block/remote-block.tsx

```diff
@@ -14,7 +14,8 @@ import { useRemoteBlock } from "./use-remote-block";
 
 type RemoteBlockProps = {
   graphCallbacks: Omit<
-    EmbedderGraphMessageCallbacks,
+    /** @todo-0.3 - Add these back */
+    EmbedderGraphMessageCallbacks<true>,
```

Comment:
This is perhaps the biggest "well not really migrated" part of this PR.

I wasn't able to just use the methods from the BP, even though we migrated a lot of their message definitions to match, this is partly [due to our messageCallbacks not being assignable](https://github.com/hashintel/hash/pull/2072/files#r1106014332), but also because some of our implementations don't actually match yet (for example `uploadFile`). I'm not sure how we go about fixing these